generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/EchoBot/README.md

```diff
@@ -14,14 +14,14 @@ This bot has been created using [Microsoft Bot Framework][1], it shows how to cr
 * Type `dotnet run`.
 
 ## Interacting With Your Bot Using the Emulator
-Launch the [Microsoft Bot Framework Emulator v4][3] and open the generated project's `.bot` file.
+Launch the [Microsoft Bot Framework Emulator v4][3] and open the `$safeprojectname$.bot` file.
 
-# Testing the bot using Bot Framework Emulator **v4**
```

Comment:
There are both version 3.x and 4.x releases of the Emulator.  I've been very prescriptive in calling out that it needs to be v4.  To make matters worse, the last release that went out was a v3 release so when you nav to the releases page, the first thing you will see is a v3 release.  I'd leave the call to action for v4 in the readme.